iped-api/src/main/java/iped/localization/Messages.java

```diff
@@ -34,6 +34,7 @@ public static ResourceBundle getExternalBundle(String bundleName, Locale locale)
         } catch (URISyntaxException e1) {
             e1.printStackTrace();
         }
+        file = new File(System.getProperty("user.dir")+ "/iped/localization");
```

Comment:
@patrickdalla what is the need for this? It is ignoring the block above...

Reply:
I think this was a patch to allow easy debugging that I mistakenly pushed. Can you remove please?